pipes/config.yaml

```diff
@@ -87,7 +87,7 @@ env_vars:
   # The directory path to the location of the GATK jar file.
   # It must be explicitly given since GATK has to be licensed and downloaded
   # manually out of band.
-  GATK_PATH: "/humgen/gsa-hpprojects/GATK/bin/GenomeAnalysisTK-3.6-0-g89b7209"
+  GATK_PATH: "/idi/sabeti-scratch/shared-resources/software/gatk/GenomeAnalysisTK-3.8-0-ge9d806836"
```

Comment:
Somewhat annoying there isn't an extracted directory for v3.8 within `/humgen/gsa-hpprojects/GATK/bin`. Should we ping someone upstairs?